station_lidar_stream: true
station_camera_frame_skip_ratio: 1
station_camera_resize_ratio: 1
station_status_stream: true
station_WIFI_ip: 172.30.87.106
print_debug: false
station_camera_stream: true
station_5G2_ip: 192.168.1.40
station_5G1_ip: 10.162.148.135
station_mode: WIFI
station_camera_jpeg_compression_ratio: 50